finesse/gui/main_window.py

```diff
@@ -40,6 +48,20 @@ def __init__(self) -> None:
         opus: QGroupBox = OPUSControl("127.0.0.1")
         layout_right.addWidget(opus)
 
+        bb_monitor: QGroupBox = BBMonitor()
+        dp9800: QGroupBox = DP9800(8)
+        tc4820_hot: QGroupBox = TC4820("hot")
+        tc4820_cold: QGroupBox = TC4820("cold")
+
+        sub_layout_right = QGridLayout()
+        sub_layout_right.addWidget(bb_monitor, 0, 0, 1, 2)
+        sub_layout_right.addWidget(dp9800, 1, 0, 1, 2)
+        sub_layout_right.addWidget(tc4820_hot, 2, 0, 1, 1)
+        sub_layout_right.addWidget(tc4820_cold, 2, 1, 1, 1)
+        sub_right = QWidget()
+        sub_right.setLayout(sub_layout_right)
+        layout_right.addWidget(sub_right)
+
```

Comment:
I don't think you need a sublayout for the right-hand side. You should be able to put all the widgets on the right side in a single `QGridLayout`, as I've done here: https://github.com/ImperialCollegeLondon/FINESSE/blob/gui_merge_for_jon/finesse/gui/main_window.py

The `opus` control will need to span two columns.